modules/dicehub/extension/action.go

```diff
@@ -49,7 +51,47 @@ type Version struct {
 	SwaggerContent []byte // content of swagger.yml
 }
 
-func (s *extensionService) InitExtension(addr string) error {
+func (s *extensionService) pushGitExtensions(gitAddr string) error {
+	dir, err := ioutil.TempDir(os.TempDir(), "*")
+	if err != nil {
+		return err
+	}
+	defer os.RemoveAll(dir)
```

Comment:
It's better to store in suitable place, and do git pull after cloned. I think you'd better add TODO to describe this plan, you can do it later.

Reply:
ok